infra/cifuzz/actions/Dockerfile

```diff
@@ -34,7 +34,8 @@ RUN add-apt-repository \
 
 RUN apt-get update && apt-get install docker-ce docker-ce-cli containerd.io -y
 
-RUN git clone -b ci-fuzz https://github.com/google/oss-fuzz.git /src/oss-fuzz
+ENV OSS_FUZZ_HOME=/src
```

Comment:
Again please don't use "home" to mean the root of the OSS-Fuzz directory.
Also, in this case, it means the parent, so please make this env var OSS_FUZZ_ROOT and make it point to /src/oss-fuzz (you'll need to update entrypoint.py too).

Reply:
Fixed.